girder/utility/server.py

```diff
@@ -190,7 +187,7 @@ def loadRouteTable(reconcileRoutes=False):
     :returns: The non empty routes (as a dict of name -> route) to be mounted by CherryPy
               during Girder's setup phase.
     """
-    pluginWebroots = plugin_utilities.getPluginWebroots()
+    pluginWebroots = {}  # plugin_utilities.getPluginWebroots()
```

Comment:
Plugin webroots is a feature not implemented yet.  It should only be a matter of adding it to the entrypoint decorator.